### ISSUE TITLE
Temperature value mixing

### ISSUE THREAD

**Issue by djuseeq** [UnifiedEngineering/T-962-improvements#223]
Hi and first of all thanks for your effort to rewrite the firmware and for sharing.
I got my oven and immediately i did the improvements what is recommended and updated the FW. 
First i used for CJ a DS1820 with one wire setup and i got strange temperature value for the right and CJ sensor.
![1820_OneWire](https://user-images.githubusercontent.com/30174351/152341280-e654af04-acd8-4f73-a4cc-8f4b18920157.jpg)

 In the serial terminal i saw the MCU has detected the DS sensor as 18S20. So i tried with 3 wire setup (separate Vcc) then i got this.
![1820_vcc](https://user-images.githubusercontent.com/30174351/152343908-5f362ed4-85a6-49fc-bded-2c26247ff2a3.jpg)

After i found a 18B20, this sensor give back a correct value but the R sensor have exact the same value as the CJ sensor. 
![18B20_vcc](https://user-images.githubusercontent.com/30174351/152344601-26d83d70-8e93-4081-9196-baaf464b2c5a.jpg)

This is the serial debug message
![18B20_log](https://user-images.githubusercontent.com/30174351/152344838-4cf6b95f-5a20-4075-b6ec-4be5c940831d.jpg)

and this is my pcb
![IMG_20220203_123453](https://user-images.githubusercontent.com/30174351/152345020-2ba06c4e-ab95-492c-8108-86195e64c0ea.jpg)

**Comment by xnk** [comment]
Hmm the fact that the R channel would show the same as the cold junction would indicate that the thermocouple amplifier is completely shorted. If you have the possibility to flash the original firmware, which temperature is that one showing? (It will assume a 20C cold junction and otherwise just take the average value of the two, so it may be difficult to spot that something is off. You haven’t touched any of the cables going to the thermocouples what it looks like, and not the trimmer potentiometers either, so I’m surprised by this zero reading.

**Comment by djuseeq** [comment]
Thanks for your reply. You were right. I have checked on the OPA left channel output i measured ~20mV while on the right channel was 0mV. I swapped the temperature probes, i got the same result so the problem is in the amplifier circuit. I checked all the resistors + capacitors in the amplifier circuit, they had the right values ( capacitors compared with caps in the left channel). I set 60 celsius and turned on the oven then the right channel has showed some value but it was too low. Then i set it to maximum feedback for the OPA right channel and i get this value
![IMG_20220211_124711](https://user-images.githubusercontent.com/30174351/153590799-cef0464e-8f6a-4590-b1d9-80228e128783.jpg)

The value of CJ has jumped up maybe because is not glued to the screw terminal yet. Now i know why you guys use an a external circuitry for the temperature amplifier, i think i will do the same ;) . Thanks
